src/scripts/viewer/2d/package2dViewController.js

```diff
@@ -54,4 +50,20 @@ module.exports = function($scope, $routeParams, $http, $location) {
     var path = getLocation($routeParams, /* is2d = */ false);
     $location.path(path);
   }
+
+  function toggleSize() {
+    $scope.showSize = !$scope.showSize;
+    refreshGraph();
```

Comment:
@anvaka I have no idea how to use Angular and it seems like there is some wrapper (`an`) around Angular that makes this even more mysterious.

How can I make this `toggleSize()` function refresh/re-render the graph?

Reply:
I don't remember most of it as well :). I think you could use approach similar to node highlighting. Check the `graphViewer.js` and `graphUI.js` files.

Reply:
Now that I think about it, I think I want to make this part of the url, because I would like to link directly to this page from https://packagephobia.now.sh 

Actually, I already have a link but I would like the link to enable this feature which is why I need it in the URL.

Thoughts?